install/streams.csv

```diff
@@ -2138,7 +2138,9 @@ THOB,22,H,N,200,true,false,false,2011-01-01T00:00:00Z,9999-01-01T00:00:00Z
 THOB,23,B,N,50,true,false,false,2011-01-01T00:00:00Z,2019-12-16T00:00:00Z
 THOB,23,H,N,200,true,false,false,2011-01-01T00:00:00Z,9999-01-01T00:00:00Z
 THPS,20,H,N,200,true,false,true,2001-11-27T00:00:00Z,2005-02-07T00:00:00Z
-THQ2,10,,,100,false,false,false,2012-12-18T01:00:00Z,9999-01-01T00:00:00Z
+THQ2,10,E,H,100,true,false,false,2012-12-18T01:00:00Z,2023-08-07T23:26:00Z
+THQ2,10,E,H,100,false,false,false,2023-08-11T02:44:00Z,2023-08-13T21:14:53Z
```

Comment:
Same with this one. Dates should be merged into the latest install with archive cleanup/renaming...